core/domain/email_subscription_services_test.py

```diff
@@ -0,0 +1,88 @@
+from core.domain import email_subscription_services
+from core.domain import subscription_services
+from core.platform import models
+from core.tests import test_utils
+import feconf
+
+(email_models,) = models.Registry.import_models([models.NAMES.email])
+(user_models,) = models.Registry.import_models([models.NAMES.user])
+
+USER_NAME = 'user'
+USER_EMAIL = 'user@test.com'
+
+USER_NAME_2 = 'user2'
+USER_EMAIL_2 = 'user2@test.com'
```

Comment:
Add all above constants in test class defined below. Don't create global constants here.